packages/connectivity/connectivity_macos/CHANGELOG.md

```diff
@@ -1,13 +1,6 @@
-## 2.0.0-nullsafety
-
-* Update version to (semi-belatedly) meet 1.0-consistency promise.
-
-## 0.2.0-nullsafety.1
+## 2.0.0
 
 * Remove placeholder Dart file.
-
-## 0.2.0-nullsafety
-
 * Update Dart SDK constraint.
```

Comment:
Probably worth being more explicit about this, and saying "Update Dart SDK constraint for compatibility with null safety" since it no longer has the context of the `-nullsafety` suffix.

Reply:
done